FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -103,7 +103,7 @@ public async Task<string> ExecuteAsync(IProgress<string> progress, CancellationT
 
                 await foreach (var input in inputs.WithCancellation(cancellationToken))
                 {
-                    while (_result.RunningJobIds.Count > _schedulerConfiguration.MaxConcurrencyCount)
+                    while (_result.RunningJobIds.Count >= _schedulerConfiguration.MaxConcurrencyCount)
```

Comment:
Question: We plan to don't allow the exactly **MaxConcurrencyCount** running jobs here?

Reply:
This is a bug:  if `_result.RunningJobIds.Count > _schedulerConfiguration.MaxConcurrencyCount` then there is `MaxConcurrencyCount ` + 1 running jobs.